src/config.rs

```diff
@@ -1189,6 +1189,31 @@ impl FontOffset {
     }
 }
 
+impl Default for FontOffset {
+    fn default() -> FontOffset {
+        FontOffset { x: 0.0, y: 0.0 }
+    }
+}
+
+/// Modifications to glyph positions within their cells
+///
+/// By default the glyphs are located at the bottom of the cell which can be
+/// undesirable. This gives the user a way to shift where the glyphs are
+/// displayed in their cells.
+#[derive(Clone, Copy, Debug, Deserialize)]
+pub struct GlyphOffset {
```

Comment:
The layout of this type is identical to `FontOffset`. Naming details aside, it would be nice to avoid that duplication. From a config file perspective (_alacritty.yml_), the user won't see the actual type, so not having a distinct type would just be an implementation detail.

Maybe we can just call it something like `Delta` instead of `FontOffset`/`GlyphOffset`. Then it can be used for the cell size change (currently the config `font.offset` object), *global* glyph offset as you've added here, and in the future, per-face glyph offset.